plugins/pagerduty/config.d.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /**
+   * Configuration for the PagerDuty plugin
+   * @visibility frontend
+   */
+  pagerduty?: {
+    /**
+     * Optional Events Base URL to override the default.
+     * @visibility frontend
+     */
+    eventsBaseUrl?: string;
```

Comment:
note: the double optional here does seem funny, in that a user could add the `pagerduty` key with no children, but rather than make eventsBaseUrl required, this should let the plugin expand over time if other optional config settings are added, leaving the eventsBaseURL still optional.